grouper/fe/settings.py

```diff
@@ -25,6 +25,7 @@ def override_timezone(self, value):
     "permission_request_text_help": None,
     "port": 8989,
     "refresh_interval": 60,
+    "shell": [["/bin/bash", "bash"], ["/bin/zsh", "zsh"]],
```

Comment:
can we make this default similar to the dev config default? so `[["/bin/false", ""]]` seems similar enough for me.